### PR TITLE
minor addition to make life easier for those with different route values

### PR DESCRIPTION
**Pull request by JonathanTech**
minor addition to make life easier for those with different route values

### CODE REVIEW

**Comment by austin-comer** [comment]
I'm not familiar with venue route values. What's the difference?

**Comment by JonathanTech** [comment]
@austin-comer the last version was assuming you were using `":venue_id/" in your route, this just makes it configurable.

**Comment by austin-comer** [comment]
Gotcha gotcha. I figured that's what it was shortly after I commented (which is why I deleted the comment) 😄

File: lib/bypass_auth.js [hunk 1]
```diff
@@ -68,12 +68,12 @@ BypassAuth.restrictToSuperAdmin = function(req, res, next) {
     next();
 };
 
-BypassAuth.restrictToVenues = function(req, res, next) {
+BypassAuth.restrictToVenuesWithRouteValue = function(routeValue){ return function(req, res, next) {
```
Comment (jspies) [inline lib/bypass_auth.js]: I would make a second method or keep the same name with optional parameter. This breaks backwards compatibility.

File: lib/bypass_auth.js [hunk 2]
```diff
@@ -82,6 +82,7 @@ BypassAuth.restrictToVenues = function(req, res, next) {
   } catch(exception) {
     return res.sendStatus(401);
   }
-};
+}};
+BypassAuth.restrictToVenues = BypassAuth.restrictToVenuesWithRouteValue('venue_id');
```
Comment (jspies) [inline lib/bypass_auth.js]: Wait, I see this now. Nvm.

Comment (JonathanTech) [reply lib/bypass_auth.js]: I was about to point you to it and wondered if I messed that up

**Pull request merged**